src/app_service/service/community/service.nim

```diff
@@ -377,6 +377,11 @@ QtObject:
         error "error: ", methodName="leaveCommunity", errDesription = "result is nil"
         return
 
+      # Update community so that joined, member list and isMember are updated
+      let updatedCommunity = response.result["communities"][0].toCommunityDto()
```

Comment:
@jrainville I guess this is here because of `communities` module? 

Just two things briefly to discuss:
- I am wondering if a logged in user leaves certain community isn't that update propagated from the `status-go` to all members of that community, I expected that info to be received via `messages.new` sent from `status-go`.  So are we sure that a community is updated appropriately in a runtime for other members when after I leave it?
- From the logical point of view, you must admit, it looks a bit weird to update a community you're about to leave. But this part will be deleted, I guess, once we update that `communities module` (as we discussed in some other PR) and get prepare the data provided by `communities module` on demand (when we really need them), that way maintaining an accurate states of community, members there and so won't be needed. This job is waiting for us. :)

Reply:
I just tested and we indeed don't receive a signal from status-go when we leave a community. I also tested with mobile leaving a community and I didn't receive a signal either, so there seems to be a problem on status-go. 
I'll open an issue in status-go to track it.